packages/snowpack/src/types/plugin-build-script.d.ts

```diff
@@ -0,0 +1,6 @@
+// TODO: remove in 3.0.0
+// note(drew): this is required because lerna build doesn’t seem to properly symlink types in snowpack.
+declare module '@snowpack/plugin-build-script' {
+  function PluginFactory(config: any, options: any): any;
+  export default PluginFactory;
+}
```

Comment:
New workaround for as long as Snowpack imports `@snowpack/plugin-build-script`: local types. Lerna really had trouble building these types and making them available for Snowpack in CI. This is a workaround that we can delete once we stop importing these packages in snowpack directly.

To be clear: it’s possible to have these types be managed in the plugin itself. However, I think the path to that involves not using `lerna run *`, and instead resorting to some Rube Goldberg machine of a build step which carefully builds everything in a particular order.